specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4197,21 +4211,21 @@
 			},
 			"context": {
 				"type": "string",
-				"description": "The context of the skill."
+				"description": "Represents the level at which operations take place, such as the document root or document content (for example, /document or /document/content)."
 			},
 			"inputs": {
 				"type": "array",
 				"items": {
 					"$ref": "#/definitions/InputFieldMappingEntry"
 				},
-				"description": "The inputs of the skill."
+				"description": "Inputs of the skills could be a column in the source data set, or the output of an upstream skill."
```

Comment:
@jeji1101 Terminology consistency: Please call it "data source" instead of "source data set".